acceptance/install/scenario3_test.go

```diff
@@ -31,11 +32,16 @@ var _ = Describe("<Scenario3> RKE, Private CA, Service", func() {
 		metallbURL = "https://raw.githubusercontent.com/google/metallb/v0.10.3/manifests/metallb.yaml"
 		localpathURL = "https://raw.githubusercontent.com/rancher/local-path-provisioner/v0.0.20/deploy/local-path-storage.yaml"
 
+		nodeIP, err := proc.RunW("kubectl", "get", "nodes",
+			"-o", `jsonpath='{.items[*].status.addresses[?(@.type=="InternalIP")].address}'`)
+		Expect(err).NotTo(HaveOccurred(), nodeIP)
+
 		flags = []string{
-			"--skip-default-namespace",
+			"--system-domain", fmt.Sprintf("%s.omg.howdoi.website", nodeIP),
```

Comment:
This seems to break the RKE test:
```
Invalid value: "epinio.'10.162.231.172'.omg.howdoi.website"
```

The IP seems to be protected by `' ... '`.

Reply:
Yes, it was the quotes. I fixed this.